### ISSUE TITLE
[source-klarna] incorrect config.region values: change 'us' --> 'na'

### ISSUE THREAD

**Issue by gndaniela** [airbytehq/airbyte#38580]
### Connector Name

source-klarna

### Connector Version

0.2.4

### What step the error happened?

During the sync

### Relevant information

Current [Klarna endpoints](https://docs.klarna.com/klarna-payments/api/payments-api/#tag/API-URLs):

- Europe:  https://api.klarna.com/
- North America:  https://api-na.klarna.com/
- Oceania:  https://api-oc.klarna.com/

The extractor fails when using 'us', it hits a wrong endpoint: `'api-us.klarna.com'`

### Relevant log output

```shell
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api-us.klarna.com', port=443): Max retries exceeded with url: /settlements/v1/payouts?size=500 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1034e6940>: Failed to resolve 'api-us.klarna.com' ([Errno 8] nodename nor servname provided, or not known)"))
```

### Contribute

- [X] Yes, I want to contribute

**Comment by marcosmarxm** [comment]
Feel free to ping me on Slack if you have questions about the contribution @gndaniela

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap.  This issue has not had any activity for 180 days, suggesting that it's not as critical as others.  It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.

**Comment by marcosmarxm** [comment]
This was solved by #38709
